packages/appsync-modelgen-plugin/src/scalars/supported-directives.ts

```diff
@@ -138,6 +138,11 @@ export const directives = /* GraphQL */ `
   # primaryKey directive
   directive @primaryKey(sortKeyFields: [String!], queryField: String) on FIELD_DEFINITION
   directive @index(name: String!, sortKeyFields: [String], queryField: String) on FIELD_DEFINITION
+
+  directive @hasOne(fields: [String!]) on FIELD_DEFINITION
```

Comment:
We could change this but I think it opens a new discussion since that's how the directive is defined in the design doc and the cli changes are done and merged now